package.json

```diff
@@ -46,16 +46,18 @@
 		"xmldom": "^0.1.27"
 	},
 	"peerDependencies": {
-		"react": "^16.11.0",
```

Comment:
Why did you downgrade React? Also, Typescript version should be bumped to the latest

Reply:
Reverted since Elements has 16.11.0 now